CONTRIBUTING.md

```diff
@@ -1,78 +1,66 @@
+## Licensing
+Any XRPL Standards document can be referred to interchangeably as an "XLS", "XRPL Standard", or "document".  Copyright on all content is subject to the terms of this [license](LICENSE), and all contributors grant everyone a royalty-free license to use their contributions, including the following grants: 
 
-## Summary
+- Copyright: a royalty-free license to anyone to use any contributions submitted to this repository.
+- Patent: a commitment to license on a royalty-free basis any essential patent claims relating to any contributions in this repository.
 
-Copyright on all content is subject to the terms of the [license](LICENSE).
+## Specification Process
 
-All contributors grant everyone:
+### 1. Start a Discussion & Gather Feedback
 
-Copyright: a royalty-free license to use the copyrights for their contributions.
-Patent: a commitment to license on a royalty-free basis their essential patent claims reading on their contributions.
+Before opening a PR with any kind of formal proposal, please first gather community input by starting a [Discussion](https://github.com/XRPLF/XRPL-Standards/discussions). Discussions are suitable for early work-in-progress: ask, suggest, add, and make sweeping changes. Collecting such feedback helps to refine your concept, and is required in order to move forward in the specification process.
 
-## Background
+#### Discussion Title
+When creating a new discussion for your idea, the discussion title should follow the naming convention `XLS-{0000}d {Title}`, where `{0000}` is a unique number for the XLS, `d` indicates that the document is a Draft (i.e., work in progress), and `{Title}` is a descriptive title for the proposed document.
 
-The work of the XRP Ledger community is open source, collaborative, and welcoming of all contributors participating in good faith. [Learn more about the XRP Ledger at XRPL.org](https://xrpl.org/).
+#### Specification Number
+Use the next number that has not yet been used. If a conflict occurs, it will be fixed by a maintainer or editor. Maintainers or editors also reserve the right to remove or re-number proposals as necessary. The number is important, as it will be used to reference features and ideas throughout the community.
 
+### 2. Closing Discussion
 
-## Process
+When a discussion has produced a well-refined standard, authors should post a comment to the discussion noting that the discussion will be closed in a few days. This allows time (for those engaged with the discussion) to submit final commentary. 
 
-The XRPL-Standards process attempts to be easy to use, but also rigorous enough that there are permalinks to revisions of documents for reference purposes.
+Once this waiting period has elapsed, the standard's author may close the discussion from further comment, and then move the discussion to a [**specification pull request**](#3.-specification-pull-requests) to add the standard into the repository as a markdown file (see below for specification formats).
```

Comment:
```suggestion
Once this waiting period has elapsed, the standard's author may close the discussion from further comment, and then move the discussion to a [**specification pull request**](#3-specification-pull-requests) to add the standard into the repository as a markdown file (see below for specification formats).
```
(Need to remove the "." from the anchor.)

Note that you can get the anchor by clicking the 3-dots menu, selecting "View file", navigating to the section you want the anchor for, and hovering your mouse just to the left of it. A link icon will appear, which you can right-click on to copy the URL. (For this section right now, it's https://github.com/XRPLF/XRPL-Standards/blob/248828e94151197f46cdc619488b9ae19f6a17db/CONTRIBUTING.md#3-specification-pull-requests) Then the URL only needs the anchor portion.

Reply:
Oh, thanks, and nice catch -- that's super helpful!